Source/santabundleservice/SNTBundleService.m

```diff
@@ -226,6 +226,9 @@ - (NSDictionary *)generateEventsFromBinaries:(NSArray *)fis
 
       MOLCodesignChecker *cs = [fi codesignCheckerWithError:NULL];
       se.signingChain = cs.certificates;
+      se.teamID = cs.teamID;
+      se.signingID = cs.signingID;
```

Comment:
You need to prefix the Signing ID with the Team ID e.g. `7Z6EMTD2C6:org.wireshark.Wireshark` and handle if this is a platform binary to be consistent with the rest of the codebase and the sync protocol docs.

If you look at MOLCodeSignChecker you'll see that this just returns [the raw signing ID](https://github.com/google/macops-molcodesignchecker/blob/5060bcc8baa90bae3b0ca705d14850328bbbec53/Source/MOLCodesignChecker/MOLCodesignChecker.m#L287-L289). 

```objc
- (NSString *)signingID {
  return self.signingInformation[(__bridge id)kSecCodeInfoIdentifier];
}
```

You need to format it like so:

https://github.com/google/santa/blob/9b184ed4fb9e4e720699b905f7b04f5c7b5cf018/Source/santactl/Commands/SNTCommandFileInfo.m#L385-L396

Reply:
I have posted a fix.